### PR TITLE
AV-38552

### PR DESCRIPTION
**Pull request by shubhamavi**

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+galaxy_info:
+  role_name: avise-vmware
```
Comment (chaitanyaavi) [inline meta/main.yml]: hyphen in the name is not as per galaxy standards lets keep this underscore we are not changing old names for backward compatibility.

**Comment by grastogi23** [comment]
let us do a demo on thursday. Also, would like see test automation plan.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,140 @@
...
+
+[![Ansible Galaxy](https://img.shields.io/badge/galaxy-avinetworks.avise__vmware-blue.svg)](https://galaxy.ansible.com/avinetworks/avise_vmware/)
+
+Using this module you are able to install the Avi Vantage Service Engine, to your system. However, minimum requirements must be met. Please visit the Avi SE Requirements webpage: https://kb.avinetworks.com/docs/latest/system-requirements-hardware/
+
+## Requirements
+
+- `avisdk` python library, which can be installed by `pip install avisdk --upgrade`
+- `pyvmomi` python library, which can be installed by  `pip install pyvmomi --upgrade`
+- `requests_toolbelt` python library, which can be installed by `pip install requests_toolbelt --upgrade`
```
Comment (grastogi23) [inline README.md]: do we need a specific version? can we make it >= xxx version

File: meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    - sdk
+    - lbaas
+    - lb
+    - virtualserver
+    - cloud
+    - loadbalancer
+    - gslb
+    - vmware
+    - vcenter
+# dependencies:
```
Comment (chaitanyaavi) [inline meta/main.yml]: Why we have this dependency commented we need this need to uncomment this

File: README.md [hunk 4]
```diff
@@ -1,2 +1,140 @@
...
+| `se_vmw_ovf_properties` | No | `None` | Other Controller ovf properties in key value format | 
+
+### VMware Deployment Example
+```
+
+---
+- hosts: controller
+  roles: 
+    - name: avinetworks.avisdk
+    - role: avinetworks.avise_vmware
```
Comment (manojkumarjain) [inline README.md]: Role parameters should be passed in vars 
roles:
  - name: ansible-role-avise-vmware
  - role: ansible-role-avise-vmware
    vars:
      se_master_ctl_ip: 10.52.3.150
      se_master_ctl_username: admin
      se_master_ctl_password:  admin@123
      se_cloud_name: Default-Cloud
      se_cluster_name: cluster-0-1
      ovftool_path: /usr/lib/vmware-ovftool
      vcenter_host: 10.10.2.7
      vcenter_user: root
      vcenter_password: vmware
      se_vmw_datacenter: Engineering
      se_vmw_cluster: Eng2
      se_vmw_ovf_networks:
        'Data Network 1': pg-17
        'Data Network 2': pg-18
        'Management': pg-mgmt
      se_vmw_vm_name: manoj-avise-vmware
      se_vmw_power_on: true
      se_vmw_vcenter_folder: manojjain
      se_vmw_number_of_cpus: 2
      se_vmw_memory: 2048

File: meta/main.yml (hunk 3 above)
Comment (manojkumarjain) [reply meta/main.yml]: Yes, playbook is not running without  sdk dependency

File: library/deploy_se.py [hunk 5]
```diff
@@ -0,0 +1,479 @@
...
+            module.exit_json(msg='A VM with the name %s updated successfully' %
+                                 (module.params['se_vmw_vm_name']), changed=True)
+        if changed and check_mode:
+            module.exit_json(changed=True)
+        else:
+            module.exit_json(
+                msg='A VM with the name %s is already present' % (
+                    module.params['se_vmw_vm_name']))
+
+    ova_file = module.params['se_vmw_ova_path'] + '/' + module.params['se_vmw_ova_image_name']
```
Comment (manojkumarjain) [inline library/deploy_se.py]: We have already computed   se_vmw_ova_path . It shdn't be recomputed here
fix as follow
#ova_file = module.params['se_vmw_ova_path'] + '/' + module.params['se_vmw_ova_image_name']
   ova_file = module.params['se_vmw_ova_path']

File: README.md (hunk 2 above)
Comment (shubhamavi) [reply README.md]: I did the task using the following versions:
pyvmomi -  6.7.1.2018.12
requests_toolbelt - 0.8.0

File: meta/main.yml (hunk 3 above)
Comment (shubhamavi) [reply meta/main.yml]: fixed and pushed

File: library/deploy_se.py (hunk 5 above)
Comment (shubhamavi) [reply library/deploy_se.py]: I intended to keep the path to the file and filename separate
so the full file path, being passed to ovftool, must include both

File: README.md (hunk 4 above)
Comment (shubhamavi) [reply README.md]: updated README

**Pull request merged**